app/ai/voice/agents/breeze_buddy/agent/__init__.py

```
@@ -597,6 +622,42 @@ async def on_pipeline_error(task, error):
+                        call_sid=self.call_sid or "",
+                        context="mid-call",
+                    )
+                except Exception as fb_err:
+                    logger.warning(f"STT fallback record_failure failed: {fb_err}")
+
+            # Alert and end call — no mid-call swap in Phase 1
+            fire_and_forget(self._send_mid_call_stt_alert())
+            try:
+                await task.queue_frames([EndFrame()])
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)